.isort.cfg

```diff
@@ -9,4 +9,4 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-known_third_party=openupgradelib,setuptools
+known_third_party = mock,openupgradelib,setuptools
```

Comment:
Why the extra space in between?

Reply:
I had deleted the entire line of 'known_third_party' and ran pre-commit run -a --show-diff-on-failure and that was how that line was automatically created (with extra spaces), anyway I already deleted the extra spaces :-)

Reply:
black doesn't format .cfg, so it's weird what you say, but let's move on.